pkgs/development/python-modules/pyrtlsdr/default.nix

```
@@ -3,32 +3,19 @@
 , fetchPypi
 , fetchpatch
 , rtl-sdr
-, m2r
+, setuptools
 }:
 
 buildPythonPackage rec {
   pname = "pyrtlsdr";
-  version = "0.2.7";
+  version = "0.2.93";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "7942fe2e7821d09206002ea7e820e694094b3f964885123eb6eee1167f39b8da";
+    sha256 = "sha256-LeKbtOQDcIinjokBK8LMhLc9xFxgYIsot9kD9ikjuiY=";
   };
 
-  # Replace pypandoc dependency by m2r
-  # See https://github.com/roger-/pyrtlsdr/pull/78
-  patches = [
-    (fetchpatch {
-      url = "${meta.homepage}/commit/2b7df0b.patch";
-      sha256 = "04h5z80969jgdgrf98b9ps56sybms09xacvmj6rwcfrmanli8rgf";
-    })
-    (fetchpatch {
-      url = "${meta.homepage}/commit/97dc3d0.patch";
-      sha256 = "1v1j0n91jwpsiam2j34yj71z4h39cvk4gi4565zgjrzsq6xr93i0";
-    })
-  ];
-
-  nativeBuildInputs = [ m2r ];
+  propagatedBuildInputs = [ setuptools ];
```

This is only needed at build time?
```suggestion
  nativeBuildInputs = [ setuptools ];
```

---

I don't really understand how this works but `setuptools` seems to be required at runtime. If I put it in `nativeBuildInputs` instead, I get this error when trying to use the library:

```
$ python myscript.py
Traceback (most recent call last):
  File "/path/to/myscript.py", line 6, in <module>
    from rtlsdr import RtlSdr
  File "/nix/store/snmxzglk7whjmlnzqw7j67s3hhy07ry7-python3-3.10.11-env/lib/python3.10/site-packages/rtlsdr/__init__.py", line 19, in <module>
    import pkg_resources
ModuleNotFoundError: No module named 'pkg_resources'
```

---

Then it is okay. I was not sure about it.